Dockerfile

```diff
@@ -38,13 +42,11 @@ RUN mkdir scripts
 COPY scripts/ scripts/
 ENV PATH="/scripts/:${PATH}"
 
-# Cache Gradle 8.2.1
-RUN mkdir gradle-cache-tmp \
-        && cd gradle-cache-tmp \
-        && wget https://services.gradle.org/distributions/gradle-8.2.1-bin.zip \
-        && unzip gradle-8.2.1-bin.zip \
-        && touch settings.gradle \
-        && gradle-8.2.1/bin/gradle wrapper --gradle-version 8.2.1 --distribution-type all \
-        && ./gradlew \
-        && cd .. \
-        && rm -rf ./gradle-cache-tmp \
+SHELL ["/bin/bash", "--login", "-c"]
+ENV NODE_VERSION v20.11.0
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash \
+    && source "$HOME/.nvm/nvm.sh" \
+    && nvm install $NODE_VERSION
+ENV PATH="//root/.nvm/versions/node/$NODE_VERSION/bin:${PATH}"
+
+RUN which npm
```

Comment:
Just curious: what made `npm` be installed in that Docker image? Is it installed as part of the `nvm` install script (`curl -o- …/nvm/v0.39.7/install.sh`)?